gpii/node_modules/flowManager/src/FlowManagerUtilities.js

```diff
@@ -184,6 +185,55 @@
         }
     });
 
+    fluid.defaults("gpii.request.flowManager.nonPrivacyFilteredMatchMakingRequest", {
+        gradeNames: ["gpii.request.flowManager.matchMakingRequest", "autoInit"],
+        listeners: {
+            onReadyToMatch: "{that}.processMatch"
+        }
+    });
+
+    // TODO: Restore the ability of the ontology manager to straightforwardly form single transforms and remove this hard-coded path
+
+    gpii.request.flowManager.privacy.privacyToSettingsTransformRules = fluid.require("${universal}/testData/ontologies/privacy-flat.json");
```

Comment:
I haven't encountered any issues with regards to the lifecycleManager web-based tests.